### PR TITLE
Add base class for simple stateless wide nodes

### PR DESCRIPTION
**Pull request by avevad**
### Changelog entry 

...

### Changelog category 

* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

**Review by igormunkin — DISMISSED** [review]
@avevad, thanks for the patch! LGTM, but please consider a couple of nits below.

File: ydb/library/yql/minikql/computation/mkql_simple_codegen.h [hunk 1]
```diff
@@ -114,6 +122,65 @@ class TSimpleStatefulWideFlowCodegeneratorNode
...
+        return DoGenGetValuesBase(ctx, statePtrVal, genToBlock);
+    }
+#endif
+};
+
+template<typename TDerived>
+class TSimpleStatelessWideFlowCodegeneratorNode
+        : public TStatelessWideFlowCodegeneratorNode<TSimpleStatelessWideFlowCodegeneratorNode<TDerived>> 
+        , public TSimpleWideFlowCodegeneratorNodeLLVMBase {
+    using TBase = TStatelessWideFlowCodegeneratorNode<TSimpleStatelessWideFlowCodegeneratorNode<TDerived>>;
```
[the remark this answers is not in the document]
Comment (avevad) [reply ydb/library/yql/minikql/computation/mkql_simple_codegen.h:137]: Looks like they are already the same 🤔

[the remark this answers is not in the document]
Comment (igormunkin) [reply ydb/library/yql/minikql/computation/mkql_simple_codegen.h:137]: PEBKAC, the wrong line.

**Review by igormunkin — APPROVED** [review]
@avevad, thanks for the fixes! LGTM.

**Pull request merged**